### ISSUE TITLE
Apply for a reprint license

### ISSUE THREAD

**Issue by Jovel-G** [Valandur/Web-API#133]
Thanks for your contribution. I’m using the plugin now, which greatly improved my management efficiency, felling awesome. May I reprint the plugin on MCBBS (a Chinese-language minecraft forum) with all copyright information retained? :)

**Comment by Valandur** [comment]
Yes of course, that would be awesome! Make sure to leave a link to the post you make here :)
